fn_apility/tests/test_fn_apility.py

```diff
@@ -0,0 +1,49 @@
+# -*- coding: utf-8 -*-
+"""Tests using pytest_resilient_circuits"""
+
+from __future__ import print_function
+import pytest
+from resilient_circuits.util import get_config_data, get_function_definition
+from resilient_circuits import SubmitTestFunction, FunctionResult
+
+PACKAGE_NAME = "fn_apility"
+FUNCTION_NAME = "fn_apility"
+
+# Read the default configuration-data section from the package
+config_data = get_config_data(PACKAGE_NAME)
+
+# Provide a simulation of the Resilient REST API (uncomment to connect to a real appliance)
+resilient_mock = "pytest_resilient_circuits.BasicResilientMock"
+
+
+def call_fn_apility_function(circuits, function_params, timeout=10):
+    # Fire a message to the function
+    evt = SubmitTestFunction("fn_apility", function_params)
+    circuits.manager.fire(evt)
+    event = circuits.watcher.wait("fn_apility_result", parent=evt, timeout=timeout)
+    assert event
+    assert isinstance(event.kwargs["result"], FunctionResult)
+    pytest.wait_for(event, "complete", True)
+    return event.kwargs["result"].value
+
+
+class TestFnApility:
+    """ Tests for the fn_apility function"""
+
+    def test_function_definition(self):
+        """ Test that the package provides customization_data that defines the function """
+        func = get_function_definition(PACKAGE_NAME, FUNCTION_NAME)
+        assert func is not None
+
+    @pytest.mark.parametrize("apility_lookup_type, apility_lookup_value, expected_results", [
```

Comment:
Is this being used? The intent is to mock out the results from the endpoint to make sure your code is processing the inputs and outputs correct. We can review if this is not clear.

Reply:
Done changes as we discussed last time.